internal/engine/dbfs/table.go

```diff
@@ -0,0 +1,42 @@
+package dbfs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+
+	"github.com/xqueries/xdb/internal/id"
+)
+
+// Table is a wrapper around the table directory of a specific table.
+type Table struct {
+	id id.ID
+	fs afero.Fs
+}
+
+// DataFile returns the data file of the table that this object represents.
```

Comment:
The data file is a paged file, meaning it consists of multiple pages. If a table has too much data to fit on a single page, it will take up multiple pages, but all in this file.
Also, for better understanding (once that is proven to work, I'll upload a concept), this is the current structure.
```
+ /
| tables
| + tables.info // contains entries of the form myTable=someID
| + someID
| | + data
| | + schema
| + otherID
| | +data
| | + schema
```